gain: 0.5
laser_min_distance: 5.2
laser_max_distance: 0.2
frequency: 50.0
